### PR TITLE
order page added

### PR DESCRIPTION
**Pull request by Deepika0788**
Hello Lokesh,

Please review the code changes for order page.

Thanks,
Lokesh Kumar

### CODE REVIEW

File: src/tests/OrderTest.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package tests;
+
+public class OrderTest {
+
+	public void orderTest() {
```
Comment (LokeshKumar1779) [inline src/tests/OrderTest.java:5]: @Test annotation is missing

File: src/pages/OrderPage.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package pages;
+
+public class OrderPage {
+
```
Comment (LokeshKumar1779) [inline src/pages/OrderPage.java]: remove unnecessary spaces

**Review by LokeshKumar1779 — CHANGES_REQUESTED** [review]
Hello Deepika,

Please check my comments and do the necessary changes.

Thanks,
Lokesh Kumar

File: src/pages/OrderPage.java (hunk 2 above)
Comment (Deepika0788) [reply src/pages/OrderPage.java]: spaces removed

**Review by LokeshKumar1779 — APPROVED** [review]
PR approved

**Comment by Deepika0788** [comment]
Order Page changes merging with master branch

**Pull request merged**